packages/framework-core/src/booster-graphql-dispatcher.ts

```diff
@@ -150,13 +156,15 @@ export class BoosterGraphQLDispatcher {
     return result
   }
 
-  private async handleDisconnect(envelope: GraphQLRequestEnvelope): Promise<void> {
-    if (!envelope.connectionID) {
+  private async handleDisconnect(connectionID?: string): Promise<void> {
+    if (!connectionID) {
       // This should be impossible, but just in case
       this.logger.debug("Received a DISCONNECT message but field 'connectionID' is missing. Doing nothing")
```

Comment:
Perhaps this should be `this.logger.error`?

Reply:
Well, it is a very rare situation (I don't think it could happen) and if it happens, nothing wrong will occur. So I think it is ok to just print it. But! I think it makes sense to change it to `this.logger.info` so that it is always printed, not only in debug mode.

Reply:
Done here: c48ee3469cf108744a2abbf9faa0ed22910b3022